src/coreclr/runtime.proj

```diff
@@ -99,8 +98,8 @@
       <_CoreClrBuildArg Include="-cmakeargs &quot;-DCLR_CMAKE_ESRP_CLIENT=$(DotNetEsrpToolPath)&quot;" />
     </ItemGroup>
 
-    <ItemGroup Condition="'$(TargetsBrowser)' == 'true'">
-      <_CoreClrBuildArg Include="-keepnativesymbols" />
+    <ItemGroup Condition="'$(TargetsBrowser)' == 'true' or '$(KeepNativeSymbols)' != 'false'">
+      <_CoreClrBuildArg Include="-cmakeargs &quot;-DCLR_CMAKE_KEEP_NATIVE_SYMBOLS=true&quot;" />
```

Comment:
What do you think of removing the logic for the `-keepnativesymbols` parameter in `build-runtime.sh` that maps it to `CLR_CMAKE_KEEP_NATIVE_SYMBOLS` if we're switching to use that everywhere?